web/src/components/organisms/Project/Schema/index.tsx

```diff
@@ -0,0 +1,169 @@
+import styled from "@emotion/styled";
+import Layout, { Header, Content } from "antd/lib/layout/layout";
+import Sider from "antd/lib/layout/Sider";
+import React, { useState } from "react";
+import { useParams, useNavigate } from "react-router-dom";
+
+import MoleculeHeader from "@reearth-cms/components/molecules/Common/Header";
+import ProjectMenu from "@reearth-cms/components/molecules/Common/projectMenu";
+import WorkspaceCreationModal from "@reearth-cms/components/molecules/Common/WorkspaceCreationModal";
+import ModelCreationModal from "@reearth-cms/components/molecules/Schema/ModelCreationModal";
+import SchemaMenu from "@reearth-cms/components/molecules/Schema/SchemaMenu";
+
+import useDashboardHooks from "../../Dashboard/hooks";
+
+import useHooks from "./hooks";
+
+export interface FormValues {
+  name: string;
+  description: string;
+}
+
+const ProjectSchema: React.FC = () => {
+  const navigate = useNavigate();
+
+  const [collapsed, setCollapsed] = useState(false);
+  //   const [selectedType, setSelectedType] = useState<FieldType | null>(null);
```

Comment:
Lots of commented out code with no explanation in the PR descirption. If they aren't needed please remove them